app/views/datasets/_datafile_table.html.erb

```diff
@@ -7,38 +7,42 @@
   </tr>
   <tbody>
     <% datafiles.each do |datafile| %>
-      <tr>
-        <td class="title small">
-          <%= link_to (datafile.name ? datafile.name : 'Data Link'),
-              datafile.url,
-              :data => {:'ga-event' => "datafile-download"}
-          %>
-        </td>
-        <% if datafile.format.blank? %>
-          <td class="dgu-unavailable">N/A</td>
-        <% else %>
-          <td><%= datafile.format.upcase %>
+    <tr class="<%= show_more?(datafiles, datafile) %> dgu-datafile">
```

Comment:
Identation was correct there. Maybe I wasn't clear, sorry: so far we've indented the html inside the <% for/if %> statements (like below). I'm happy to discuss changing that, but we should be consistent